openfisca_core/scenarios.py

```diff
@@ -714,20 +714,20 @@ def json_or_python_to_test(value, state = None):
 
 def iter_over_entity_members(entity_description, scenario_entity):
         # One by one, yield individu_role, individy_legacy_role, individu_id
```

Comment:
comment should have fixed indent too

Reply:
Thanks @Br3nda, I'm puzzled as why the linter didn't catch that, as it seems to be in flagrant violation of E113.